src/components/ebay-skeleton-loader/index.marko

```diff
@@ -0,0 +1,48 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+static var ignoredAttributes = [
+    "a11yText",
+    "class",
+    "size",
+    "type",
+    "multiline",
+    "count",
+    "columns",
+    "isVisible",
+];
+static var validTypes = ["avatar", "button", "text", "form", "image", "tile"];
+static var validSizes = ["large", "small"];
+$ var size = validSizes.includes(input.size) ? input.size : "default";
+$ var type = validTypes.includes(input.type) ? input.type : "button";
```

Comment:
I'm not sure how I feel about defaulting to `"button"`, is that the most likely object to be making a skeleton for?

Reply:
The idea was to restrict to one of these types from ebay UI side. And picked button as random. We can also not load a skeleton if it doesn't match any type.

Reply:
I would imagine that a skeleton without a type just adds `skeleton` to the class list, and nothing else

Reply:
Agreed. Unless its broken then we shouldn't default to that

Reply:
As discussed, now defaulted to form as we do not have standalone `skeleton` class.